homeassistant/helpers/translation.py

```diff
@@ -528,6 +529,34 @@ def async_translations_loaded(hass: HomeAssistant, components: set[str]) -> bool
     )
 
 
+@callback
+def async_get_exception_message(
+    translation_domain: str | None,
+    translation_key: str | None,
+    translation_placeholders: dict[str, str] | None = None,
+    language: str = "en",
```

Comment:
I think we should not add this parameter since it's not used. If needed, we can add it in the future.